databus-client2/src/main/java/com/bazaarvoice/emodb/databus/client2/factory/DatabusFactory.java

```diff
@@ -0,0 +1,59 @@
+package com.bazaarvoice.emodb.databus.client2.factory;
+
+
+import com.bazaarvoice.emodb.common.jersey2.Jersey2EmoClient;
+import com.bazaarvoice.emodb.databus.client2.client.DatabusClient;
+import com.bazaarvoice.emodb.databus.client2.discovery.DatabusDiscovery;
+import com.google.common.base.Throwables;
+import com.google.common.util.concurrent.Service;
+import org.glassfish.jersey.client.ClientConfig;
+import org.glassfish.jersey.client.ClientProperties;
+import org.glassfish.jersey.client.JerseyClient;
+import org.glassfish.jersey.client.JerseyClientBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.Serializable;
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Factory for creating {@link DatabusClient} clients.
+ */
+public class DatabusFactory implements Serializable {
+
+    private final Logger _log = LoggerFactory.getLogger(DatabusFactory.class);
+
+    private final DatabusDiscovery.Builder _databusDiscoveryBuilder;
+    private final String _apiKey;
+
+    public DatabusFactory(DatabusDiscovery.Builder databusDiscoveryBuilder, String apiKey) {
```

Comment:
consider changing signature to something like
```suggestion
    public DatabusFactory(DatabusDiscovery databusDiscovery, String apiKey) {
```